packages/eslint-config-react/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## 0.0.5 - 2019-09-19
+- Update eslint packages
```

Comment:
I appreciate that we have release notes for these two packages now.  Can we link to them in the README files?